### DISCUSSION TITLE
Sensori no mosquitto, quali?

### DISCUSSION THREAD

**Discussion by mriitan** [indomus/forum#1406] [HUB: Home Assistant]
Viste le difficoltà che ho nel riuscire a collegare correttamente i sensori shelly, volevo sapere se esiste un prodotto che si integri facilmente senza troppo caos e che rilevi temepratura e umidità di una stanza.
Grazie
mr

**Comment by XRedShark** [comment]
Cc2531 zigbee con antenna e ti si apre il mondo. Google e trovi molte idee.

**Reply by snakuzzo** [reply]
Magari CC2531 no...che è un chip obsoleto da tempo. CC2562 meglio...ultimo chip che trovi sia sulla zbee duo del nostro @gio-dot che sul nuovo dongle usb di itead.
La CC2531 ormai la prendi solo se devi testare qualcosa o se devi usarla come sniffer (sempre in ottica test e sviluppo)

**Comment by snakuzzo** [comment]
Perdonami, ma se non sai niente di mqtt  sai che esiste già l'integrazione nativa e devi fare solo un click?
